pkg/internal/checkup/trex/client.go

```diff
@@ -32,3 +46,111 @@ func NewClient(vmiSerialClient vmiSerialConsoleClient, namespace string, verbose
 		verbosePrintsEnabled: verbosePrintsEnabled,
 	}
 }
+
+func (t Client) WaitTrexServerReady(ctx context.Context, vmiName string) error {
+	const (
+		interval = 5 * time.Second
+		timeout  = time.Minute
+	)
+	var err error
+	ctxWithNewDeadline, cancel := context.WithTimeout(ctx, timeout)
+	defer cancel()
+	conditionFn := func(ctx context.Context) (bool, error) {
```

Comment:
nit: Please add a new line here.

Reply:
not sure where exactly you want the new line..